Document new one-step setup for FM (#931)
neo4j/docs-aura: modules/ROOT/pages/fleet-management/setup.adoc

Reviewer: No really sure how to improve that but this step is only needed for "manual registration". And not relevant for the one-step setup. Maybe naming it "Manually register the deployment" and adding a sentence about that?

Author: I can increase the heading level so that it falls under `Setup using plugin (Neo4j 4.4.x, 5.26.x, <2026.03.0)` rather than being a sibling. Though these steps still can be used as a fallback for version with the built-in, I think it may be a reasonable simplification?

= Add a deployment
:description: This page describes how to add a self-managed deployment.

> Adding a self-managed deployment to Fleet Manager is a straightforward process. With newer versions of Neo4j, connect to your deployment from the wizard, and we will take care of the rest.

Reviewer 2: ```suggestion
Adding a self-managed deployment to Fleet Manager is a straightforward process. 
With newer versions of Neo4j, all you need to do is connect to your deployment from the wizard and the rest is done for you.
```

The wizard provides two top-level options.

* *Monitored* - this option allows you to set up Fleet Manager.
* *Unmonitored* - this option allows you to add an unmonitored deployment that can still be connected to using the provided connection URL.
//add a link here to Manage instances and the new section about remote connections.

== One-step setup (Neo4j 2026.03.0 and newer)

For Neo4j version 2026.03.0 and newer, Fleet Manager is built-in and can be enabled in one step.

Select *Monitored* and click the *Connect...* button to connect to your instance. In the dialog that appears, input the URL and credentials to your deployment and connect.

[.shadow]
.Connect to your deployment
image::connectionmodalnonremote.png[]

> After connecting, we will run a procedure call to check whether your deployment meets the requirements for Fleet Manager. The results are clearly communicated using a banner.

Reviewer 2: ```suggestion
After connecting, a procedure call runs to check whether your deployment meets the requirements for Fleet Manager. 
The results are clearly communicated using a banner.
```

[.shadow]
.Ready for Fleet Manager
image::fm-connectionsuccess.png[]

Finally, select *Next* to have Aura automatically generate and add an access token to your deployment.

> Your deployment is now ready to use with Fleet Manager!

Reviewer 2: ```suggestion
Your deployment is now ready to use with Fleet Manager.
```

The wizard provides two top-level options.

* *Monitored* - this option allows you to set up Fleet Manager.
* *Unmonitored* - this option allows you to add an unmonitored deployment that can still be connected to using the provided connection URL.
//add a link here to Manage instances and the new section about remote connections.

== One-step setup (Neo4j 2026.03.0 and newer)

For Neo4j version 2026.03.0 and newer, Fleet Manager is built-in and can be enabled in one step.

Select *Monitored* and click the *Connect...* button to connect to your instance. In the dialog that appears, input the URL and credentials to your deployment and connect.

[.shadow]
.Connect to your deployment
image::connectionmodalnonremote.png[]

After connecting, a procedure call runs to check whether your deployment meets the requirements for Fleet Manager. 
The results are clearly communicated using a banner.

[.shadow]
.Ready for Fleet Manager
image::fm-connectionsuccess.png[]

Finally, select *Next* to have Aura automatically generate and add an access token to your deployment.

Your deployment is now ready to use with Fleet Manager.

> == Setup using plugin (Neo4j 4.4.x, 5.26.x, <2026.03.0)

Reviewer: ```suggestion
[[_setup_using_plugin_neo4j_4_4_x_5_26_x_2026_03_0]]
== Setup using plugin (Neo4j 4.4.x, 5.26.x, <2026.03.0)
```

according to how this is (mostly) done elsewhere (for example see `[[parameter-data-types]]` in this repo). Otherwise at least the IDE is complaining.

Author: I can see a lot of headings that don't have an alias. I think it's only needed if we cross-reference it from another page?

Reviewer: I thing is would still be good in case if someone changes the text of the header - not sure if we'd get build errors in that case. Using a separate anchor decouples the text from the anchor.

Author: Ofc, we *do* cross-reference it 🤦  Will add it!
